src/Tools/GenerateProps/targetgroups.props

```diff
@@ -77,29 +70,24 @@
       <Imports>netcoreapp1.0</Imports>
       <CompatibleWith>netstandard1.7</CompatibleWith>
     </TargetGroups>
-    <TargetGroups Include="dnxcore50">
-      <ConfigurationErrorMsg>$(ConfigurationErrorMsg);DNXCore50 has been deprecated.  Please use NETStandard1.X or NETCoreApp1.0 instead.</ConfigurationErrorMsg>
-    </TargetGroups>
-    <TargetGroups Include="net463">
-      <PackageTargetFramework>net463</PackageTargetFramework>
-      <TargetingPackNugetPackageId>Microsoft.TargetingPack.NETFramework.v4.6.3</TargetingPackNugetPackageId>
-      <NuGetTargetMoniker>.NETFramework,Version=v4.6.3</NuGetTargetMoniker>
-      <Imports>net462</Imports>
+    <TargetGroups Include="netcoreapp">
+      <PackageTargetFramework>netcoreapp1.1</PackageTargetFramework>
+      <NuGetTargetMoniker>.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
+      <Imports>netcoreapp1.0</Imports>
```

Comment:
Should this import `netcoreapp1.1` so that configurations like `netcoreapp1.1` will be considered compatible with `netcoreapp`?  We could consider doing the reverse as well (I handle cycles).

Same is true for the other versionless targetgroups you've added.